.gitignore

```diff
@@ -1,7 +1,8 @@
 node_modules
 npm-debug.log
+package-lock.json
```

Comment:
This project uses yarn (there's a yarn.lock file in the repo), so package-lock.json shouldn't have to be ignored I think :)

Reply:
well, I was about to accidentally add it cause I used `npm install` instead of `yarn` it's a minor addition I can send it in another PR.
As a yarn project, you _should_ ignore the package-lock.json so it does not get accidentally added by unsuspecting devs 😢

Reply:
Ah okay, that makes sense. It's always a bit confusing to know when to use yarn and when to use npm honestly. Would be nice if npm and yarn would detect if a lock file from the other tool is present and warn the user about it 😅